pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -287,6 +288,36 @@ Avo will not override any events that are only defined in RudderStack, or events
   style={{ margin: '32px 0 -32px 0' }}
 />
 
+## <a name="snowplow"></a> Snowplow Data Structures
+
+### Configure the Snowplow integration
+
+To enable the direct integration with Snowplow, you need to configure it with the following parameters.
+
+- **Snowplow Organization ID**: Your organization ID issued by Snowplow.
+- **Snowplow Vendor Name**: The vendor name of your Snowplow schemas.
+- **Snowplow Access Token**: The secret access token to update your Snowplow data structures.
+
+### Publishing to Snowplow
+
+You can publish your Avo tracking plan into Snowplow with the publishing integration.
+
+> Data structures interfaces are only compatible with pipelines that have been upgraded to use Iglu Server registries, rather than static S3 registries. Please check here in the console, to see if you need an upgrade or if your registries are ready to go.
```

Comment:
```suggestion
> Data structures interfaces are only compatible with pipelines that have been upgraded to use Iglu Server registries, rather than static S3 registries. Please check in your [Snowplow console](https://console.snowplowanalytics.com/data-structures/), to see if you need an upgrade or if your registries are ready to go.
```